recipe/meta.yaml

```diff
@@ -1,18 +1,20 @@
 {% set name = "kfp" %}
-{% set version = "1.2.0" %}
+{% set version = "1.3.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e3e127d8159c7d7e798611ba889dddd581f14ea934af6ae0a703f6351e34200d
+  sha256: 04aca6204fd6f4ce0895ab1320d08a11e20ad4fb89718c514be49e3ece7dd0fe
 
 build:
   noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script:
+    - {{ PYTHON }} -m pip install kfp-pipeline-spec==0.1.4 -vv
```

Comment:
@maximerihouey -- I guess that would have been too easy. It looks like conda's pip resolver does not agree with my hack :-\

```
...
Ignoring indexes: https://pypi.org/simple
...
0 location(s) to search for versions of kfp-pipeline-spec:
Given no hashes to check 0 links for project 'kfp-pipeline-spec': discarding no candidates
ERROR: Could not find a version that satisfies the requirement kfp-pipeline-spec==0.1.4
ERROR: No matching distribution found for kfp-pipeline-spec==0.1.4
```

I will make another attempt at introducing the `kfp-pipeline-spec` feedstock that does not depend on a `.tar.gz` file from PyPI. But it might take me a little while to get a chance.